TAO/tao/Transport.cpp

```diff
@@ -164,6 +163,8 @@ TAO_Transport::TAO_Transport (CORBA::ULong tag,
 #endif /* TAO_HAS_TRANSPORT_CURRENT == 1 */
   , flush_in_post_open_ (false)
 {
+  cache_map_entry_.entry_ = 0;
+  cache_map_entry_.int_id_ = 0;
```

Comment:
I would assume that by default a platform has it, only platforms that really lack it should get a ACE_LACKS_STD_PARTIAL_SORT